backend/src/test/java/gov/cdc/usds/simplereport/api/queue/QueueMutationResolverTest.java

```diff
@@ -0,0 +1,95 @@
+package gov.cdc.usds.simplereport.api.queue;
+
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import gov.cdc.usds.simplereport.db.model.DeviceSpecimenType;
+import gov.cdc.usds.simplereport.db.model.DeviceType;
+import gov.cdc.usds.simplereport.db.model.Organization;
+import gov.cdc.usds.simplereport.db.model.auxiliary.DiseaseResult;
+import gov.cdc.usds.simplereport.db.model.auxiliary.TestResult;
+import gov.cdc.usds.simplereport.service.BaseServiceTest;
+import gov.cdc.usds.simplereport.service.DeviceTypeService;
+import gov.cdc.usds.simplereport.service.PersonService;
+import gov.cdc.usds.simplereport.service.TestOrderService;
+import gov.cdc.usds.simplereport.test_util.SliceTestConfiguration;
+import gov.cdc.usds.simplereport.test_util.TestDataFactory;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+import java.util.UUID;
+import lombok.SneakyThrows;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+
+@SliceTestConfiguration.WithSimpleReportStandardUser
+class QueueMutationResolverTest extends BaseServiceTest<PersonService> {
+  @Autowired private TestDataFactory _dataFactory;
```

Comment:
Best way to move away from `@Autowired` pattern here? 🤔

Reply:
You should be able to do something like: 

```
private TestDataFactory _dataFactory;

QueueMutationResolverTest(TestDataFactory dataFactory) {
this._dataFactory = dataFactory;
}
```
There, we're declaring the _dataFactory variable first, then creating a default constructor for the QueueMutationResolverTest. Spring looks for the default constructor and is smart enough to inject an instance of TestDataFactory when it's requested. Functionally it's not much different from `@Autowired`, since Spring is handling the dependency injection in both cases, but constructor injection is recommended by Spring. This StackOverflow and the linked blog post have more detail, if you're interested: https://stackoverflow.com/a/40620318

Reply:
Emma and I looked at this together and concluded we are not able to adapt this pattern in test files otherwise we run into this error `No ParameterResolver registered for parameter`